src/wp-includes/revision.php

```diff
@@ -665,9 +665,11 @@ function wp_get_post_revisions( $post = 0, $args = null ) {
 	}
 
 	$defaults = array(
-		'order'         => 'DESC',
-		'orderby'       => 'date ID',
-		'check_enabled' => true,
+		'order'                  => 'DESC',
+		'orderby'                => 'date ID',
+		'check_enabled'          => true,
+		'update_post_meta_cache' => false,
+		'update_post_term_cache' => false,
```

Comment:
For posts with footnotes enabled the footnotes meta data is included in the revisions (see `src/wp-includes/blocks/footnotes.php`) so I think the meta cache will need to remain primed.

You may be able to check if the post type supports footnotes.

Not priming the term cache makes sense though.

Reply:
@peterwilsoncc Footnotes - thats a good point, thanks. But still, `wp_list_post_revisions()` doesn't need to use any data from postmeta. I am thinking, as long as post information is primed, that is all it needs here.

Reply:
@waviaei Footnotes are included on the revision screen. Plugin developers can also register meta data with revisions and may choose to display that on the revisions screen too.

![Screenshot 2024-09-12 at 3 12 31 PM](https://github.com/user-attachments/assets/4578effc-81e0-460c-bf9d-7c186797302f)